frontend/src/app/features/web-node/web-node-file-upload/web-node-file-upload.component.html

```diff
@@ -68,18 +68,17 @@ <h2>Set Up Your Web Node</h2>
     }
     <div class="buttons-container fx-row-vert-cent flex-between">
       <div class="develop-buttons fx-row-vert-cent">
-        <button class="btn-aware h-lg border-rad-8 f-500 pl-12 pr-12"
-                [tooltip]="'Start a develop Block Producer Web Node'"
-                (click)="onStartDevelopWebnode()">Start BP
-        </button>
-        <button class="btn-aware h-lg border-rad-8 f-500 pl-12 pr-12"
-                [tooltip]="'Start a develop Web Node without Block Producer feature'"
-                (click)="onStartDevelopWebnodeNonBP()">Start Non BP
-        </button>
+        <!--        <button class="btn-aware h-lg border-rad-8 f-500 pl-12 pr-12"-->
```

Comment:
@directcuteo don't forget the other environments, this should be configurable, not fully disabled. If we deploy the latest version to the main web it will break.